test/CadlRanchProjects.Tests/type-property-optionality.cs

```diff
@@ -590,5 +589,90 @@ public void PlainTime()
             var output = PlainTimeProperty.DeserializePlainTimeProperty(document.RootElement);
             Assert.AreEqual(PlainTimeData, output.Property);
         }
+
+        [Test]
+        public void NotRequiredNullablePropertiesOmitedByDefault()
```

Comment:
we should use `Optional` instead of `NotRequired`

Reply:
fixed